deployments/curity/apigateway/openresty/Dockerfile

```diff
@@ -11,18 +11,14 @@ RUN git config --global url."https://".insteadOf git:// && \
 #
 # Install an example CORS plugin 
 #
-COPY openresty/cors.lua \
-     openresty/cors-*.rockspec \
-     /tmp/cors/
+COPY resources/examples/* /tmp/cors/
```

Comment:
The customer download package now has an example CORS module that the SPA uses. Customers can use their own CORS module instead if they prefer.